pig/test-domains/main.ion

```diff
@@ -0,0 +1,30 @@
+/*
+
+This file is used to test the `(import <relative-path>)` statement.
+
+Two primary things are tested:
+
+- Cyclic imports are tolerated without exception.
+- Relative paths, which are always relative to the directory containing the type universe currently being parsed.
+
+.
+├── domains
```

Comment:
Could also include the `include-missing-file.ion` domain including a missing file.

Reply:
So, yes, I could... but I think that might be confusing since it's not imported by `main.ion`.  I'm inclined to leave it out of this description.

Reply:
meh, I'll add it if only to say it's not imported by `main.ion`